File: ReverseArray/ReverseArray.py
# Reverse Array:


def ReverseArray(n, arr):
    start = 0
    end = n - 1
    while start < end:
        arr[start], arr[end] = arr[end], arr[start]
        start += 1
        end -= 1
    print(f"ReverseArray = {arr}", end=" ")


n = int(input("Enter number: "))
arr = list(map(int, input("Enter values: ").split()))
print(f"array = {arr}", end=" ")
ReverseArray(n, arr)

File: ReverseArray/ReverseArray.cs
// Reverse Array: 

using System;

class Program {
    static void ReverseArray(int n, int[] arr) {
        int start = 0, end = n - 1;

        while (start < end) {
            arr[start] ^= arr[end];
            arr[end] ^= arr[start];
            arr[start] ^= arr[end];
            start++;
            end--;
        }

        Console.Write("Reverse Array = ");
        for (int i = 0; i < n; i++) {
            Console.Write(arr[i] + " ");
        }
    }

    static void Main() {
        Console.Write("Enter number: ");
        int n = int.Parse(Console.ReadLine());

        int[] arr = new int[n];
        Console.Write("Enter values: ");
        string[] input = Console.ReadLine().Split();
        for (int i = 0; i < n; i++) {
            arr[i] = int.Parse(input[i]);
        }

        Console.Write("Array = ");
        for (int i = 0; i < n; i++) {
            Console.Write(arr[i] + " ");
        }

        ReverseArray(n, arr);
    }
}
